spec/chingu/network_spec.rb

```diff
@@ -71,7 +71,12 @@ def data_set
         expect(@client).to receive(:on_connection_refused)
 
         @client.connect
-        5.times { @client.update }
+        5.times do
+          @client.update
+          # We need to use a large timeout value here to make this spec pass on
+          # Windows. https://stackoverflow.com/a/19441512
+          sleep 0.5
```

Comment:
@Chadowo It's likely that this makes the spec run longer than before on other platforms. I don't want to get too deep into Chingu's networking code [because it looks like Wrath is the only game that uses it](https://github.com/search?q=Chingu%3A%3AGameStates%3A%3ANetworkClient&type=code), and Wrath will remain broken even with a fixed Chingu gem because it relies on Ashton as well.

Reply:
Oh, I stand corrected, it does not use Ashton, but Texplay.

Reply:
I see, thanks for the help!